src/calculateRentalCost.js

```diff
@@ -27,8 +27,24 @@
  * @return {number}
  */
 
+const STANDARD_PRICE = 40;
+const SHORT_TERM = 3;
+const MID_TERM_DISCOUNT = 20;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  const STANDARD_TERM = days * STANDARD_PRICE;
```

Comment:
And this is just an internal variable that you use inside the function. It's not supposed to 'mimic' env variable.
This one should follow standard naming camelCase convention.

Don't perceive this as a jest, at this point it's just really important for you to grab the basics.